CHANGELOG.txt

```diff
@@ -1,8 +1,22 @@
 = `gcovr` Release History and Change Log =
 
 === 3.4 ''(TODO)'' ===
- - Fix to support filtering with absolute paths
- - Added --html-encoding command line option (#139)
+ - Added --html-encoding command line option (#139).
+ - Added --fail-under-line and --fail-under-branch options,
+   which will error under a given minimum coverage. (#173, #116)
+ - Better pathname resolution heuristics for --use-gcov-file. (#146)
+ - The --root option defaults to current directory '.'.
+ - Improved reports for "(", ")", ";" lines.
+ - HTML reports show full timestamp, not just date. (#165)
+ - HTML reports treat 0/0 coverage as NaN, not 100% or 0%. (#105, #149, #196)
+ - Only Python 2.6+ is supported, with 2.7+ or 3.4+ recommended. (#195)
+ - Added CI testing for Windows using Appveyor. (#189, #200)
+ - Reports use forward slashes in paths, even on Windows. (#200)
+ - Fix to support filtering with absolute paths.
+ - Fix HTML generation with Python 3. (#168, #182, #163)
+ - Fix --html-details under Windows. (#157)
+ - Fix filters under Windows. (#158)
+
```

Comment:
You're missing these ones if I'm not mistaken:
`Fix verbose output when using existing gcov files (#143, #144)`
`Add support for coverage-04.dtd Cobertura format (#164)`

Otherwise seems good

Reply:
fixed! Thanks for spotting that :)